### ISSUE TITLE
ContextBlock ignores contents of items property

### ISSUE THREAD

**Issue by odub** [Abjad/abjad#488]
Observe:
```
>>> b = lilypondfiletools.Block()
>>> b.items.append('foo')
>>> f(b)
\score {
    foo
}
```

The contents of `b.items` are rendered into the output.

Compare:
```
>>> c = lilypondfiletools.ContextBlock()
>>> c.items.append('foo')
>>> f(c)
\context {
    \Voice
}
```

The `'foo'` string gets ignored.

ContextBlock overrides the `_format_pieces` method from Block (parent class).

Compare:
https://github.com/Abjad/abjad/blob/master/abjad/tools/lilypondfiletools/Block.py#L97-L150
https://github.com/Abjad/abjad/blob/master/abjad/tools/lilypondfiletools/ContextBlock.py#L71-L117

The `_format_pieces` method in ContextBlock makes no reference to `self.items`

**Comment by odub** [comment]
First attempt at a fix is here:
https://github.com/Abjad/abjad/pull/489

Pending what Travis says, it works, but I'm keen nonetheless to discuss if it's the best solution.

I'm not keen on `else pass` (copied from block) but maybe error handling should go into another clean up issue covering both this and other similar corners in the code base.

**Comment by josiah-wolf-oberholtzer** [comment]
1) Can you provide a musically relevant example so I can understand the use-case? Something that could go into `ContextBlock`'s docstring example?

2) I think it's OK to not touch the `else pass` branch here. Let's talk in a second issue about cleaning that up across both Block and ContextBlock.

The formatting code for these two classes is getting sufficiently lengthy that it's probably worth breaking out chunks into private methods.

**Comment by odub** [comment]
Partial answer for now, on 1):

My case was trying to Abjadify this: https://gist.github.com/odub/a8e685f612ff51391b48#file-flatglisssnippet-ly-L110 (code from an example on the LilyPond mailing list a year or so ago)

I've actually just found a way around my problem by putting the overrides that the custom function `flatGlissOn` references into the layout block directly.

Now the problem is that I don't know enough LilyPond to know another common case needing to put arbitrary strings into a `\context` block. Maybe you do or there is one, though.

I see two choices and don't have the information to choose: either (a) take back the proposed changes and go further to remove `items` for the ContextBlock class, or (b) use a different arbitrary string example and leave the changes in. I'm happy to go with either. Probably @josiah-wolf-oberholtzer, you have a clearer view of the purpose behind `items` than I do.

**Comment by odub** [comment]
Actually, maybe for (b) the example could just be: "You can also define context block contents like this" and then a common override as a string. Seems a little like encouraging a hack over the neater Python model from the API, though.

**Comment by josiah-wolf-oberholtzer** [comment]
OK. Your use-case makes sense. You've defined a function somewhere and are then executing it in a context block. Perfectly reasonable.

I was just looking for a way to integrate the changes you're proposing into the docstring that would correspond to something you could do in LilyPond out of the box. And I think I've found the answer:

```
\layout {
    \context {
        \Staff
        \accidentalStyle dodecaphonic
    }
}
```

... which would correspond to something like ```c.items.append(r'\accidentalStyle dodecaphonic').```

Of course you could also turn the above into a LilyPondCommand and append that to the item list, but allowing arbitrary strings is universally helpful. Abjad is, after all, basically just a big string processor.

Integrate ```...items.append(r'\accidentalStyle dodecaphonic')``` into the `ContextBlock` docstring, and we should be good to go for this pull request.

**Comment by odub** [comment]
Oh, nice! Yeah, I'll just do that, much better example…

**Comment by josiah-wolf-oberholtzer** [comment]
Fixed in 343ad427d37538de8f86bce7a71d813548d9f374.
